kadena-beta/docs/Kadena-README.md

```diff
@@ -708,4 +720,4 @@ Endpoints:
   node3: 127.0.0.1:8003
 ```
 
-(c) Kadena 2017
+(c) Kuro 2017
```

Comment:
Is this correct?

Reply:
No, leave it as Kadena and update to 2020, or just ditch this line, copyright isn't neccessary now

Reply:
changed it to Kadena 2020